.github/workflows/release_package.yml

```diff
@@ -37,37 +37,78 @@ jobs:
       - name: Get branch name
         id: branch-name
         uses: tj-actions/branch-names@v5.4
-      
+
+      - name: Quick build of Empirica CLI to ensure compilation is working
```

Comment:
The problem with this step here is that it is slow. It takes ~8min. Ideally, we'd only build for linux adm64. It's ok for testing, but it's something to keep in mind.

Reply:
Yeah, will use only the amd64 once everything else works